serverless/experimental/gcp-log/README.md

```diff
@@ -21,26 +21,25 @@ Both functions are HTTP-triggered and run when their respective endpoints are re
 ### GCF function deployment:
 1.  Generate a set of public and private keys following
     [these](https://github.com/google/trillian-examples/tree/master/serverless#generating-keys)
-    instructions and set them as the `SERVERLESS_LOG_PUBLIC_KEY` and
-    `SERVERLESS_LOG_PRIVATE_KEY`.
+    instructions and set them as the `PUBLIC_KEY` and `PRIVATE_KEY`.
 1. Set `PROJECT_NAME` as the name of your GCP project (string, not number).
 1.  Deploy the Integrate function:
     ```
     gcloud functions deploy integrate \
     --entry-point Integrate \
     --runtime go116 \
     --trigger-http \
-    --set-env-vars GCP_PROJECT=${PROJECT_NAME} SERVERLESS_LOG_PUBLIC_KEY=${PUBLIC_KEY} SERVERLESS_LOG_PRIVATE_KEY=${PRIVATE_KEY} \
+    --set-env-vars GCP_PROJECT=${PROJECT_NAME},SERVERLESS_LOG_PUBLIC_KEY=${PUBLIC_KEY},SERVERLESS_LOG_PRIVATE_KEY=${PRIVATE_KEY} \
```

Comment:
nit: is it worth wrapping this whole mega-arg in double quotes to avoid confusing errors in the case where one of the env vars (probably erroneously) contains a space?

Reply:
Done.